client/lib/store-transactions/index.js

```diff
@@ -90,7 +90,9 @@ TransactionFlow.prototype._paymentHandlers = {
 	},
 
 	'WPCOM_Billing_MoneyPress_Paygate': function() {
-		var validation = validateCardDetails( this._initialData.payment.newCardDetails );
+		const newCardDetails = this._initialData.payment.newCardDetails,
+			validation = validateCardDetails( newCardDetails );
+
```

Comment:
Absolutely not mandatory but you could use destructuring here:

```javascript
const { newCardDetails } = this._initialData.payment,
  validation = validateCardDetails( newCardDetails );
```

This saves a few characters :P.